AVFoundation-Combine/Video Player/VideoPlayerView.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  VideoPlayerView.swift
+//  AVFoundation-Combine
+//
+//  Created by Juan Carlos Ospina Gonzalez on 02/08/2020.
+//  Copyright © 2020 József Vesza. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+final class VideoPlayerView: UIView {
```

Comment:
I feel that this view could use better naming. In the view controller, you referred to it as `customUI`, while here it's a video player. It represents a view that holds controls, and will be added to the content overlay... so how about something like `VideoPlayerContentOverlay`?